content/docs/dvclive/get-started.md

```diff
@@ -89,31 +82,33 @@ more details and other supported frameworks.
 Once you have added DVCLive to our python code, you can run the script as you
 would usually do:
 
-```dvc
+```cli
 $ python train.py
 ```
 
-DVCLive will generate a [report](/doc/dvclive/outputs#report) in
+DVCLive will generate a [report](/doc/dvclive/api-reference/live/make_report) in
 `dvclive/report.html` containing all the logged data. It will be automatically
```

Comment:
Definitely not a blocker, but do we have an issue for showing other report types (including vs code and studio) here? I know @shcheklein mentioned it but not sure we noted it anywhere in GitHub.

Reply:
The thing for this report is that it was moved up to showcase value before entering in DVC integration, showing the other reports before the DVC integration could be misleading (for now)?